website/templates/emails/notify_base.mako

```diff
@@ -43,7 +43,7 @@
                 <tbody>
                     <tr>
                         <td style="border-collapse: collapse;">
-                            <p class="small text-center" style="text-align: center;font-size: 12px;">To change your email notification preferences, visit your project or user settings.</p>
+                            <p class="small text-center" style="text-align: center;font-size: 12px;">To change your email notification preferences, visit your project or <a href="http://osf.io/settings">user settings</a>.</p>
```

Comment:
Shouldn't this link to `/settings/notifications/`? Also, can you avoid hardcoding the domain, so that this will work on staging, localhost, etc.?